apps/mark-scan/frontend/src/components/live_check_button.tsx

```diff
@@ -0,0 +1,45 @@
+/* istanbul ignore file */
```

Comment:
A copy of https://github.com/votingworks/vxsuite/blob/main/apps/admin/frontend/src/components/live_check_button.tsx and https://github.com/votingworks/vxsuite/blob/main/apps/scan/frontend/src/components/live_check_button.tsx

Will consolidate in a future PR